diff --git i/third_party/google-styleguide/pylintrc w/third_party/google-styleguide/pylintrc
index c36c30c..878e253 100644
--- i/third_party/google-styleguide/pylintrc
+++ w/third_party/google-styleguide/pylintrc
@@ -132,6 +134,7 @@ disable=apply-builtin,
         useless-else-on-loop,
         useless-suppression,
         using-cmp-argument,
+        wrong-import-position,  # https://github.com/PyCQA/pylint/issues/3349
         xrange-builtin,
         zip-builtin-not-iterating,
 
