diff --git i/third_party/google-styleguide/pylintrc w/third_party/google-styleguide/pylintrc
index 807e8f3..8192708 100644
--- i/third_party/google-styleguide/pylintrc
+++ w/third_party/google-styleguide/pylintrc
@@ -308,7 +308,7 @@ max-module-lines=99999
 # spaces.  Google's externaly-published style guide says 4, consistent with
 # PEP 8.  Here, we use 2 spaces, for conformity with many open-sourced Google
 # projects (like TensorFlow).
-indent-string='  '
+indent-string='    '
 
 # Number of spaces of indent required inside a hanging  or continued line.
 indent-after-paren=4
